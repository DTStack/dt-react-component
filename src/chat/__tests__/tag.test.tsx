import React from 'react';
import { render } from '@testing-library/react';

import Tag from '../tag';

describe('Test Chat Tag', () => {
    it('Match Snapshots', () => {
        expect(render(<Tag className="test">Tag</Tag>).asFragment()).toMatchSnapshot();
    });
});
