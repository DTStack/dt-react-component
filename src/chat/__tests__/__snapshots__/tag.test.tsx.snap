// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Chat Tag Match Snapshots 1`] = `
<DocumentFragment>
  <div
    class="dtc__tag test"
  >
    <span
      class="dtc__icon dtc__tag__icon"
    >
      <svg
        fill="none"
        height="1em"
        viewBox="0 0 6 6"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 3C0 1.34315 1.34315 0 3 0C4.65685 0 6 1.34315 6 3C6 4.65685 4.65685 6 3 6C1.34315 6 0 4.65685 0 3Z"
          fill="url(#paint0_linear_5253_12831)"
        />
        <circle
          cx="3"
          cy="3"
          fill="url(#paint1_linear_5253_12831)"
          r="3"
        />
        <defs>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="paint0_linear_5253_12831"
            x1="0"
            x2="7.2238"
            y1="0"
            y2="2.60057"
          >
            <stop
              stop-color="#08C4FF"
            />
            <stop
              offset="0.5"
              stop-color="#4892FF"
            />
            <stop
              offset="1"
              stop-color="#8A61FF"
            />
          </lineargradient>
          <lineargradient
            gradientUnits="userSpaceOnUse"
            id="paint1_linear_5253_12831"
            x1="0"
            x2="7.2238"
            y1="0"
            y2="2.60057"
          >
            <stop
              stop-color="#08C4FF"
            />
            <stop
              offset="0.5"
              stop-color="#4892FF"
            />
            <stop
              offset="1"
              stop-color="#8A61FF"
            />
          </lineargradient>
        </defs>
      </svg>
    </span>
    Tag
  </div>
</DocumentFragment>
`;
